src/arithmetic.jl

```diff
@@ -129,6 +128,16 @@ function remainder_product(a, b, aux, Δnegl)
     Δ = Δnegl + Δb * a.rem + Δa * b.rem + a.rem * b.rem
     return Δ
 end
+function remainder_product(a::TaylorModel1{TaylorN{T}, S},
+                           b::TaylorModel1{TaylorN{T}, S},
+                           auxT, Δnegl) where {T, S}
+    N = get_numvars()    
+    auxQ = IntervalBox(-1 .. 1, Val(N))
```

Comment:
Could you add somewhere a comment indicating that this method assumes that the domain (for the `TaylorN`  variables) is the symmetric normalized box?

Reply:
Bump...

Reply:
Can you add a comment pointing out that the domain of  validity assumed for the `TaylorN` is the  symmetric box?